appium/webdriver/switch_to.py

```diff
@@ -22,7 +22,7 @@
 
 
 class MobileSwitchTo(SwitchTo):
-    def context(self, context_name: str) -> T:
+    def context(self, context_name: Optional[str]) -> T:
```

Comment:
why context name is optional?

Reply:
It comes from https://github.com/appium/python-client/blob/8eb4c3f7c2cb95ee81eb9c664de265edf473d9dc/test/functional/android/context_switching_tests.py#L49

Reply:
it makes sense to mention that in the docstring (e.g. passing None is equal to switch to native context)